css-shadow-parts-1/Overview.bs

```diff
@@ -7,7 +7,7 @@ Status: ED
 Work Status: exploring
 URL: http://drafts.csswg.org/css-shadow-parts/
 Editor: Tab Atkins-Bittner, Google, http://xanthir.com/contact/, w3cid 42199
-Abstract: This specification defines the ''::part()'' and ''::theme()'' pseudo-elements on <a>shadow hosts</a>, allowing <a>shadow hosts</a> to selectively expose chosen elements from their <a>shadow tree</a> to the outside page for styling purposes.
```

Comment:
Can you explain why we want to take these links out?

Reply:
That shouldn't be happening. I think I took the wrong base for this PR. I will fix that.